src/nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -517,13 +534,12 @@ def _get_primary_config(self, group_size: int) -> WeightCompressionConfig:
             codebook_values=codebook_values,
         )
 
-    def _set_weight_compression_config(
+    def apply_mixed_precision(
```

Comment:
Please update the description of this method as well. Add info about the mixed precision